packages/manager/modules/pci/src/projects/project/project.controller.js

```diff
@@ -51,6 +52,17 @@ export default class ProjectController {
     );
 
     this.PciProject.setProjectInfo(this.project);
+
+    this.vouchersCreditInfo(this.project_id).then((vouchersCreditInfos) => {
+      this.credits = vouchersCreditInfos.map((voucher) => {
+        return {
+          voucher: voucher.voucher,
+          description: voucher.description,
+          balance: voucher.available_credit.text,
+          expirationDate: moment(voucher.validity.to).format('LLL'),
+        };
+      });
+    });
```

Comment:
it's better to put this into routing deps resolve.

Reply:
Done.